packages/core/src/wallet.ts

```diff
@@ -302,14 +304,18 @@ export abstract class Wallet<T> {
   async createAllowedAddress(params: {
     address: string;
     label: string;
+    coinId: number | null;
```

Comment:
If it feels optional, how about putting it last?

Reply:
ok